spec/models/mou_signature_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+RSpec.describe MouSignature, type: :model do
+  let(:mou_signature) { build :mou_signature }
+
+  it "has a valid factory" do
+    expect(mou_signature).to be_valid
+  end
+
+  it "is invalid without a user" do
+    error_message = I18n.t("errors.messages.required")
+
+    mou_signature.user = nil
+    expect(mou_signature).not_to be_valid
```

Comment:
```suggestion
    expect(mou_signature).to be_invalid
```
I find it easier to read and it matches the spec description of "it is invalid without a user"